jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -265,7 +245,8 @@ def get_image_data_iterator(app, file_obj, data_label, ext=None, try_gwcs_to_fit
 
     # load Roman 2D datamodels:
     elif HAS_ROMAN_DATAMODELS and isinstance(file_obj, rdd.DataModel):
-        data_iter = _roman_asdf_2d_to_glue_data(file_obj, data_label, ext=ext)
+        data_iter = _roman_asdf_2d_to_glue_data(file_obj, data_label, ext=ext,
+                                                try_gwcs_to_fits_sip=try_gwcs_to_fits_sip)
```

Comment:
This should no longer be reachable by Imviz, so I think we can leave this out for now since other places it could _possibly_ be reached (mosviz and cubeviz) don't have orientation plugin anyways.